src/test/java/org/hydev/mcpm/utils/PairTest.java

```diff
@@ -0,0 +1,39 @@
+package org.hydev.mcpm.utils;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+
+/**
+ * Tests for Pair class
+ */
+class PairTest
+{
+    @Test
+    void gettersSetters()
+    {
+        var p = new Pair<>("a", 1);
+
+        assert p.k().equals(p.getKey());
+        assert p.k().equals("a");
+        assert p.v().equals(p.getValue());
+        assert p.v().equals(1);
+
+        try
+        {
+            p.setValue(2);
+            assert false;
+        }
+        catch (Pair.UnmodifiableException ignored) { }
+    }
```

Comment:
This looks like its own test case again.

Reply:
done